packages/client/.env

```diff
@@ -1,2 +1,3 @@
-REACT_APP_HTTP_URL="http://localhost:3008"
 PORT=3007
+REACT_APP_HTTP_URL="http://localhost:3008"
+REACT_APP_SOCKET_URL="http://localhost:3008"
```

Comment:
doesn't this need another protocol? like ws:// or something like that?

Reply:
Good point, but I am a bit confused on this at the moment... Just put it a such because of documentation here: https://socket.io/docs/v3/client-api/index.html

And currently the browser does:
![image](https://user-images.githubusercontent.com/908935/100498894-5acc7380-315d-11eb-919b-de42ceb4496b.png)